docs/advanced_configuration.md

```diff
@@ -2,6 +2,35 @@
 
 If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [React Native RUM setup documentation][2].
 
+## Testing with Jest
```

Comment:
This part is needed as soon as you've done the basic set-up, before going further in implementation.
Customers will want to make sure their tests are passing before going implementing more features, so they can test their implementation along the way. 
I think it will also be more useful than each individual guides for custom instrumentations and initializations.